client/signup/jetpack-connect/index.jsx

```diff
@@ -440,9 +445,9 @@ const JetpackConnectMain = React.createClass( {
 		}
 		return this.renderSiteEntry();
 	}
-} );
+}
 
-export default connect(
+const connectComponent = connect(
 	state => {
 		const getJetpackSite = ( url ) => {
```

Comment:
It's worth to mention that we try to avoid these in `mapStateToProps` at all costs as they're not performant. But it's outside of this PR's scope of course 😉